qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -18,6 +26,12 @@ You can download an example sample information file and prep information file fr
 Sample information file
 -----------------------
 
+Similar to a `QIIME metadata file, <http://qiime.org/documentation/file_formats.html#metadata-mapping-files>`__
```

Comment:
This description is a bit of and can lead to more confusion. One of the main problems that we are facing when communicating the idea of Sample Information file is that the users think that this file is the QIIME mapping file. However, it is conceptually different, since the QIIME metadata file contains information from the Sample Information file and the Prep information file. For example, in the QIIME metadata file there is the BarcodeSequence column. That would be conceptually incorrect to add in the Sample  Information file. So I would suggest not mentioning the QIIME mapping file at all in here.